dali/pipeline/pipeline.cc

```diff
@@ -767,6 +767,15 @@ OpNode * Pipeline::GetOperatorNode(const std::string& name) {
   return &(graph_.Node(name));
 }
 
+const OpNode *Pipeline::GetInputOperatorNode(const std::string& name) {
+  for (auto &[op_name, node] : input_operators_) {
+    if (op_name == name) {
+      return node;
+    }
+  }
+  DALI_FAIL("Could not find an input operator with name " + name);
```

Comment:
Add quotes for readability.
```suggestion
  DALI_FAIL(make_string("Could not find an input operator with name \"", name, "\"");
```